project/src/main/control-settings.gd

```diff
@@ -0,0 +1,191 @@
+class_name ControlSettings
```

Comment:
Consider changing this to `KeybindSettings` to avoid ambiguity with UI controls.

Reply:
Additionally, this class should be in the `src/main/keybind` package.

Reply:
I've moved this class into the `src/main/keybind` package and renamed it to `KeybindSettings`.

I've changed all other variable/class/method names centered around "player controls" to use the word keybind as well.